src/cmdstan-guide/optimize_config.Rmd

```diff
@@ -26,10 +26,27 @@ of default configuration options:
 #         tol_rel_grad = 10000000 (Default)
 #         tol_param = 1e-08 (Default)
 #         history_size = 5 (Default)
+#     jacobian = 0 (Default)
 #     iter = 2000 (Default)
 #     save_iterations = 0 (Default)
 ```
 
+## Jacobian adjustments
+
+The `jacobian` argument specifies whether or not the call to the model's
+log probability function should include
+the log absolute Jacobian determinant of inverse parameter transforms.
+Without the Jacobian adjustment, optimization
+returns the maxiumum likelihood estimate (MLE),
```

Comment:
-> the (regularized) maximum likelihood estimate (MLE)

(as prior terms are still included in the target)